apps/system/js/operator_variant_handler.js

```diff
@@ -270,7 +270,8 @@
           'ril.data.passwd': 'password',
           'ril.data.httpProxyHost': 'proxy',
           'ril.data.httpProxyPort': 'port',
-          'ril.data.authtype': 'authtype'
+          'ril.data.authtype': 'authtype',
+          'ril.data.mtu': 'mtu'
```

Comment:
the value is missing in related test https://github.com/mozilla-b2g/gaia/blob/master/apps/system/test/unit/operator_variant_handler_test.js#L100